make/builder-shell.py

```
@@ -0,0 +1,131 @@
+
+def command(args: Namespace):
+    image = f"ghcr.io/eeems/arkes-builder:{cast(str, args.branch)}"
+    if not image_exists(image, False, True):
+        if not image_exists(image, True, True):
+            print(f"{image} does not exist")
+            sys.exit(1)
+
+        pull(image)
+
+    master_fd, slave_fd = pty.openpty()
+    proc = subprocess.Popen(
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)